### ISSUE TITLE
"Group movies into collections" cannot be disabled

### ISSUE THREAD

**Issue by BoboceaOctavian** [jellyfin/jellycon#322]
**Describe the bug**
If there are movies grouped into collections, Jellycon will always display them as a single item in the movies library, even if "Group movies into collections" is set to false.

**To Reproduce**
1. Add at least two movies into the same collection.
2. Open the movies library in Jellycon.
3. Check that the movies are displayed in a single collection item that can be opened to list the movies.
4. Go into the Jellycon settings, in the Interface tab, Item layout section and set "Group movies into collections" to false. Save and restart.
5. Open again the movies library in Jellycon.
6. Check that the movies are still displayed in a single collection item and not as separate entries.

**Expected behavior**
If the setting "Group movies into collections" is disabled, movies belonging to collections should be shown as their own entries in the libraries displayed by Jellycon.

**System (please complete the following information):**
 - OS: Android TV 11
 - Jellyfin Version: 10.8.13
 - Kodi Version: 21.0-RC1
 - Addon Version: 0.8.0+py3

**Additional context**
Collections have been created automatically with the TMDb Box Sets plugin.

I have disabled the setting with the same name in the Jellyfin server (Libraries - Display - Group movies into collections) and the "Automatically add to collection" setting for my movies library and also refreshed it. After this both the Jellyfin web UI and the Jellyfin Android app show the movies as specific entries in the library and the collections appear only in their specific "Collections" library.

**Comment by Shadowghost** [comment]
I am unable to reproduce this. I tested it with multipled accounts on different boxes and the setting works as intended.

**Comment by BoboceaOctavian** [comment]
Yeah, apologies, I kinda forgot about this report. I'm no longer sure if this is a bug / what is causing it / it is even related to JellyCon.

The idea is that if you have collections created by the TMDb Box Sets plugin these will always be shown as a single item in a library, no matter how many movies there are in it and there seems to be no way to disable this. I was looking for a behavior like this: in a "Collections" library have collections as groups that can be opened and in a "Movies" library have the movies shown as individual entries, even if they are part of a collection.

I have since removed all collections from my setup, so I am closing this bug.
